compiler/rustc_const_eval/src/check_consts/qualifs.rs

```diff
@@ -140,6 +134,7 @@ impl Qualif for HasMutInterior {
     }
 }
 
+// FIXME(const_trait_impl): Get rid of this!
```

Comment:
Well, only once we stabilized const-drop. Before that we need this qualif to ensure that stable const fn don't rely on const drop.